.eslintignore

```diff
@@ -1,5 +1,3 @@
 # Contains the PDF file of the Tag as JSON string, thus does not need to be linted
+
```

Comment:
1. Please revert all changes to this file. 
2. It is out of scope of the issue. 
3. The changes impede our ability to have the code quality we require as a project